.test.mk

```diff
@@ -74,7 +77,8 @@ test-release-asan: CMAKE_PARAMS = -DCMAKE_BUILD_TYPE=RelWithDebInfo \
                                   -DENABLE_WERROR=ON \
                                   -DENABLE_ASAN=ON \
                                   -DENABLE_UB_SANITIZER=ON \
-                                  -DENABLE_FUZZER=ON
+                                  -DENABLE_FUZZER=ON \
+                                  -DABORT_ON_LEAK=ON
```

Comment:
We may add more options like this in future, in which case we'd like to avoid patching `.test.mk`. Let's introduce a new meta-option `TEST_BUILD`, which would incorporate all similar options (currently, it'd just enable `ABORT_ON_LEAK`).

Reply:
Done.